### PR TITLE
docs: update crates & add achievements

### PR DESCRIPTION
**Pull request by konal1019**

### CODE REVIEW

File: economy/crates.md [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
 
 ## crate odds
 
 you can view the odds / chances for items from crates [here](https://github.com/mxz7/nypsi-odds)
 
 ## how do i open my crates?
 
-use the `/opencrates` command to open multiple crates at once. you can use `/use` to open specific
-crates.
+You can use the `$use <name of crate> <amount>` commmand to open specific number of a specific crate.
```
Comment (mxz7) [inline economy/crates.md]: lowercase

File: economy/crates.md [hunk 2]
```diff
@@ -2,19 +2,19 @@
 
 ## how do i get a crate?
 
-you can easily obtain vote crates by voting for nypsi with `$vote`. you can also get crates from
-tasks, daily, or the [online shop](https://ko-fi.com/tekoh/shop)
+you can easily obtain vote crates by voting for nypsi with `$vote`. You can also get crates from
```
Comment (mxz7) [inline economy/crates.md]: lowercase

**Review by mxz7 — CHANGES_REQUESTED** [review]
everything gotta be lowercase!

**Comment by mxz7** [comment]
Are you going to rectify the issues?

File: economy/crates.md (hunk 2 above)
Comment (404qua) [reply economy/crates.md]: Ok I will fix it

**Comment by mxz7** [comment]
You must resolve the conflicts

File: economy/achievements.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# achievements
+achievements are goals that when completed, give you some rewards. 
+
+## how can i see achievements?
+you can use the ` $achievements ` command to view your unlocked achievements and your progress on them. if you want to know about a particular achievement, you can use:
+```
+$achievements view < name of achievement >
+```
+this will show the details of the achievement as well as the rewards you get for completing them. there are currently 34 unique qchievements in nypsi with each having 5 levels amounting to a total of 170 achievements. you can get a list of all achievements using `$achievements all`.
```
Comment (mxz7) [inline economy/achievements.md]: for achievements specifically i'd ask you tell people to use the slash (/) command versions. it's easier for achievements, especially the find view command

**Review by mxz7 — CHANGES_REQUESTED** [review]

**Pull request merged**